generator/README.md

```diff
@@ -8,28 +8,113 @@ See installation instructions [here](./docs/GeneratorSetup.md).
 
 ## Command line
 
-The command line has two commands. Usage instructions for a command can be requested by calling it without arguments - eg, `dg generate`.
+You must have JAVA v1.8 installed (it can be [downloaded here](https://www.java.com/en/download/manual.jsp)) to be able to run the generator. Once this is installed you can run the generator by running the following command:
 
-### `generate`
+`java -jar <path to JAR file> <command> [options] <arguments>`
+
+* `<command>` one of the commands described below, `generate`, `generateTestCases`, and so on
+* `[options]` optionally a combination of options to configure how the command operates
+* `<arguments>` required inputs for the command to operate
+
+### For example
+* `java -jar generator.jar generate profile.json profile.csv`
+* `java -jar generator.jar generateTestCases profile.json violated-data-files/`
+
+### __Command:__ `generate [options] <profile path> <output path>`
 
 Generates data to a specified endpoint.
 
-### `generateTestCases`
+* `<profile path>`, a path to the profile JSON file
+* `<output path>`, a file path to where the data should be emitted to. This will be a UTF-8 encoded CSV file.
+
+#### Options
+Options are optional and case-insensitive
```

Comment:
They are optional, but you should document what their default value is - e.g. what is the default generation mode?

Reply:
For completeness this is documented, but may need to be clearer.

> Emit <generationType> data. Options are: INTERESTING (default)

See [Generator README](https://github.com/ScottLogic/data-engineering-generator/blob/master/generator/README.md)